test/integration/bench/test_bench.py

```diff
@@ -0,0 +1,44 @@
+import logging
+import pytest
+from utils import (get_admin_connection_params, get_log_lines,
+                   run_command_and_get_output)
+from subprocess import Popen, PIPE, STDOUT
+from threading import Thread
+import signal
+import os
+
+def consume_lines(port, pipe):
+    logger = logging.getLogger('localhost:{}'.format(port))
+    with pipe:
+        for line in iter(pipe.readline, b''):
+            logger.warning(line.rstrip().decode('utf-8'))
```

Comment:
I'm not sure, but ``.decode()`` uses utf-8 by default.

Reply:
In any case, explicit pointing is better